apisix/admin/v3_adapter.lua

```diff
@@ -183,6 +183,9 @@ function _M.filter(body)
 
     pagination(body, args)
     filter(body, args)
+
+    -- recalculate the amount of filtered data
+    body.count = body.list and #body.list or 0
```

Comment:
But I think the count should be the total object of the resource. Not the count of the list.

Reply:
ping @starsz @soulbird

Yes, as I mentioned in the issue, I was thinking the same thing, do you think it would be better to use `count` or add another `total` field?